src/man/include/debug_levels.xml

```diff
@@ -100,6 +100,7 @@
         introduced in 1.7.0.
     </para>
     <para>
-        <emphasis>Default</emphasis>: 0
+        <emphasis>Default</emphasis>: 0x0070 (i.e. fatal, critical and serious
+        failures)
```

Comment:
Do you think we should also mention that this corresponds to `2` ?

Reply:
Done